packages/workers-shared/asset-server-worker/src/index.ts

```diff
@@ -1,38 +1,66 @@
+import { WorkerEntrypoint } from "cloudflare:workers";
 import { AssetsManifest } from "./assets-manifest";
-import { NoopAssetsManifest } from "./assets-manifest-no-op";
+import {
+	InternalServerErrorResponse,
+	MethodNotAllowedResponse,
+	NotFoundResponse,
+	OkResponse,
+} from "./responses";
+import { getAdditionalHeaders, getMergedHeaders } from "./utils/headers";
+import { getAssetWithMetadataFromKV } from "./utils/kv";
 
-interface Env {
-	ASSETS_MANIFEST: ArrayBuffer;
-	ASSETS_KV_NAMESPACE: KVNamespace;
-}
+export default class extends WorkerEntrypoint<Env> {
+	async fetch(request: Request) {
+		if (!request.method.match(/^(get)$/i)) {
```

Comment:
This seems a bit complicated... why not:

```suggestion
		if (request.method.toLowerCase() !== "get") {
```